translations/consolinno-energy.de.ts

```diff
@@ -2454,12 +2454,12 @@ fügen Sie eine weitere hinzu.</translation>
     </message>
     <message>
         <source>CHANGENOTIFICATION_PLACEHOLDER</source>
-        <translation type="unfinished">&lt;h3&gt;Die App wurde aktualisiert&lt;/h3&gt;
-            &lt;p&gt;Version 1.7.0 (30. November 2024)&lt;/p&gt;
-            &lt;h4&gt;Neue Funktionen&lt;/h4&gt;
+        <translation>&lt;h3&gt;Die App wurde aktualisiert&lt;/h3&gt;
+            &lt;p&gt;Version 1.7.1 (11. Dezember 2024)&lt;/p&gt;
+            &lt;h4&gt;Verbesserungen&lt;/h4&gt;
```

Comment:
```suggestion
            &lt;p&gt;Version 1.8.0 (2025)&lt;/p&gt;
            &lt;h4&gt;Neue Funktionen&lt;/h4&gt;
```
Man könnte hier schon mal für das Changelog 1.8.0 vorbereiten.